types/schemas/tockb_cell_data.mol

```diff
@@ -23,10 +23,25 @@ table BtcExtra {
 }
 
 table EthExtra {
-	dummy: Bytes,
+
 }
 
 struct ToCKBTypeArgs {
 	xchain_kind: byte,
 	cell_id: OutPoint,
 }
+
+table EthCellData {
+    headers: Chain,
+}
+
+table Chain {
+    main: Bytes2,
+    uncle: Bytes2,
+}
+
+table HeaderInfo {
+    header: Bytes,
+    total_difficulty: Uint64,
+    hash: Byte32,
+}
```

Comment:
better move to eth_headers_cell_data.mol

Reply:
nod